### PR TITLE
CHANGELOG + liveupdate editable readme

### PR DESCRIPTION
**Pull request by mnordsletten**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Changelog
+
+## v0.15.0 (pre-release)
+
+- Updates to workflow
+  - All changes to the workflow have been documented in the [readme](https://github.com/includeos/IncludeOS/blob/dev/README.md)
```
Comment (MagnusS) [inline CHANGELOG]: Link is to README on this branch - could you make it relative?

Comment (mnordsletten) [reply CHANGELOG]: I've made the link relative. And I've made a lot more changes, mainly following the guidelines here: https://keepachangelog.com/en/1.0.0/
- [unreleased] section at the top
- Subsections like added, changed, removed
- made it an `.md` file so github knows to format the file like markdown
- All headers are now linkable.

File: lib/LiveUpdate/README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+```
+cd <IncludeOS src>/lib/LiveUpdate
+```
+2. Remove the version variable from the `conanfile.py`.
+```
+awk '!/version = conan_tools/' conanfile.py > temp && mv temp conanfile.py
+```
+3. Create a `build` directory in the LiveUpdate directory.
+```
+mkdir build
```
Comment (KristianJerpetjon) [inline lib/LiveUpdate/README.md]: If we do all this from the actual source folder that should be easier.

File: lib/LiveUpdate/README.md [hunk 3]
```diff
@@ -1 +1,28 @@
...
+```
+4. Add editable package with correct layout
+```
+conan editable add . liveupdate/9.9.9@includeos/latest --layout layout.txt
+```
+5. Run Conan install
+```
+conan install -if build .
```
Comment (KristianJerpetjon) [inline lib/LiveUpdate/README.md]: this creates the build folder afaik

Comment (KristianJerpetjon) [reply lib/LiveUpdate/README.md]: conan install lib/liveudate -if lib/liveupdate/build perhaps

File: lib/LiveUpdate/README.md [hunk 4]
```diff
@@ -1 +1,28 @@
...
+```
+awk '!/version = conan_tools/' conanfile.py > temp && mv temp conanfile.py
+```
+3. Create a `build` directory in the LiveUpdate directory.
+```
+mkdir build
+```
+4. Add editable package with correct layout
+```
+conan editable add . liveupdate/9.9.9@includeos/latest --layout layout.txt
```
Comment (KristianJerpetjon) [inline lib/LiveUpdate/README.md]: here we could also do it from the root folder

**Comment by mnordsletten** [comment]
@KristianJerpetjon I changed it to use root as reference instead. Looks pretty good now I think

**Pull request merged**